types/operation.generated.go

```diff
@@ -274,6 +316,47 @@ func (s UnimplementedFetcher) FetchWithContext(ctx context.Context, path string,
 	return
 }
 
+// Operation names in HTTPSigner.
+var (
+	// OpHTTPSignerQuerySignHTTP is the operation name for QuerySignHTTP in HTTPSigner.
+	OpHTTPSignerQuerySignHTTP = "QuerySignHTTP"
+)
+
+// HTTPSigner is the interface for Signer.
+type HTTPSigner interface {
```

Comment:
`http` is considered as a common initialisms and is converted to `HTTP` in interface name and operation name. Should I update GSP for consistency?

Reply:
LGTM